Makefile

```diff
@@ -62,7 +62,7 @@ build-dirs:
 test: build-dirs
 	hack/test.sh
 
-verifiy:
+verify:
```

Comment:
:mindblown:

Reply:
Yeah I was all set to write it and then I saw it was already done.... 😂